gallery/qml/IconLine.qml

```diff
@@ -0,0 +1,44 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Controls.Basic
+
+import "../app/qml/components"
+import "../"
+
+Column {
+  id: root
+
+  required property string name
+  required property url source
+  required property bool showRect
+
+  Row {
+    Rectangle { width: 100 * __dp; height: 50 * __dp
+      Image { source: root.source; anchors.verticalCenter: parent.verticalCenter
+        Rectangle { anchors.fill: parent; color: "#ff8888"; z: -1 }
+      }
+    }
```

Comment:
do not understand what this pink box is? I would remove it and add there instead one white variant of icon